DAL/contentInspectionApi.js

```diff
@@ -48,7 +54,10 @@ function isKnownGifDomain(url) {
 
     if (domain.hostname.endsWith("gfycat.com"))
         return true;
-
+    
+    if (domain.hostname.endsWith("discordapp.com"))
```

Comment:
This will flag every embed uploaded to Discord regardless of type.  The gif domain check is effectively "ignore the type, if it comes from this domain, it should be treated as a gif".  You can see it in action here: https://github.com/shatindle/discord-gif-cop/blob/6ab1ebe1b417b48437796a78206c0feb9571ed70/DAL/contentInspectionApi.js#L57